dbms/src/Interpreters/getQueryAliases.h

```diff
@@ -8,6 +8,26 @@ namespace DB
 
 using Aliases = std::unordered_map<String, ASTPtr>;
 
-void getQueryAliases(ASTPtr & ast, Aliases & aliases, int ignore_levels = 0);
+class QueryAliasesVisitor
```

Comment:
Rename the file to `QueryAliasesVisitor` ?

Reply:
Yap